src/ts/components/core/combobox/Select.tsx

```diff
@@ -42,6 +42,10 @@ interface Props
     hiddenInputProps?: object;
     /** Props passed down to the underlying `ScrollArea` component in the dropdown */
     scrollAreaProps?: ScrollAreaProps;
+    /** If set, the highlighted option is selected when the input loses focus @default `false` */
+    autoSelectOnBlur?: boolean;
```

Comment:
Is this one important enough to include a test for? That said when I try it on their [docs site](https://mantine.dev/core/select/#autoselectonblur) it doesn't seem to do what it says 🤷

Reply:
Oh.  I just answered a different question. I thought you were asking about the `clearSearchOnFocus` which is the new feature added in this PR.

Their  `autoSelectOnBlur` is not currently working as they describe in their docs.  (It's a bug.)   I reported it and they said it will be fixed in  8.2.5.  I'll wait for that before releasing